src/vm/ceeload.h

```diff
@@ -3686,6 +3686,7 @@ class ReflectionModule : public Module
     // If true, then only other transient modules can depend on this module.
     bool m_fIsTransient;
 
+#if !defined DACCESS_COMPILE && !defined CROSSGEN_COMPILE
```

Comment:
It is possible some of the changes are no longer necessary because I added the gc-sections support for removing dead code after some of the code changes. I'll go threw a few of them and be sure, although in general it's a good idea anyway to keep the #ifdefs in sync IMO